src/databases/Silo/avtSiloFileFormat.C

```diff
@@ -5464,25 +5468,41 @@ avtSiloFileFormat::FindStandardConnectivity(DBfile *dbfile, int &ndomains,
     if (!packed_conn_info) // use standard connectivity info
     {
         debug1 << "avtSiloFileFormat: using standard connectivity info" <<endl;
-        for (int j = 0 ; j < ndomains ; j++)
+
+        // Look for absence of `Domain_0/BlockNum` along with the
+        // existence of `Domains_BlockNums` & `Domain_Extents`
+        // to identify the rare "6th" silo decomp format.
+        //
+        // (yes, Domain_Extents is singular not plural form)
+        //
+        // This case is a blend of some of the arrays used for the mmadj and
+        // "standard" format flavors.
+        //
+        // It came to town in a file that has decomp dirs but the blocks
+        // are all independent and neighborless.
+        // 
+        // mmadj can't represent this case (all relationships can't be empty,
+        // in a mmadj -- its simply too dire of a state for the mmadj to face.)
+        //
+        // CYRUS NOTE: This might not actually be the "6th" format
+        // (it's at least the 5th :-), but the name conveys multitude of
+        // formats, and helps us reflect on our collective decomp decisions.
```

Comment:
Honesltly, I am not feeling all that reflective. Might be the SPF-50 I am wearing 😉